web/modules/custom/dpl_admin/dpl_admin.module

```diff
@@ -103,6 +109,36 @@ function _dpl_admin_preprocess_views_view_field_event_admin_eventseries_id(array
   $variables['output'] = Markup::create($markup);
 }
 
+/**
+ * Preprocess the author field in the "content" admin view.
```

Comment:
I couldn't see any visual indication that an Eventseries/instance was locked, when it had been locked. 

If content-lock is also applied to events, i think this should also be the case for the event admin view. 

Content from the "content" admin view will surely be edited more, but i think if there is a visual indication for that view, the same should apply if content-lock as used elsewhere.